src/BinSkim.Sdk/CompilerDataLogger.cs

```diff
@@ -75,12 +79,18 @@
~
~        public static void Initialize(string instrumentationKey)
         {
             if (s_telemetryConfiguration == null && s_telemetryClient == null)
             {
-                s_sessionId = Guid.NewGuid().ToString();
-                s_telemetryConfiguration = new TelemetryConfiguration(instrumentationKey);
```

Comment:
>s_syncRoot [](http://example.com/codeflow?start=22&length=10)

we should synchronize the telemetry creation to create one telemetry and one session